index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/style.css" />
+    <title>Coding Quiz</title>
+</head>
+<body>
+ <header>
+  <section class="head">
+    <h1>&lt Coding Quiz &gt</Coding></h1>
+  </section>
+  <section class="card-body">
```

Comment:
Would it be more semantic to use a `main` element here?

Reply:
Yes I will add a main

Reply:
I think what I meant was "Replace the `section` with a `main` element. Besides: what does the `card-body` do? It's not the body of a card, but the wrapper of all card sections, isn't it?

Reply:
Oh okay, now I get it